Makefile

```diff
@@ -144,15 +144,19 @@ ifeq ($(BUILD_IN_CONTAINER),true)
 
 client/build/index.html: $(shell find client/app -type f) $(SCOPE_UI_BUILD_UPTODATE)
 	mkdir -p client/build
-	$(SUDO) docker run $(RM) $(RUN_FLAGS) -v $(shell pwd)/client/app:/home/weave/app \
-		-v $(shell pwd)/client/build:/home/weave/build \
-		$(SCOPE_UI_BUILD_IMAGE) yarn run build
+	test "true" = "$(SCOPE_SKIP_UI_ASSETS)" || { \
```

Comment:
I'd personally find `test "true" != "$(SCOPE_SKIP_UI_ASSETS)" && { \` (or even `if test "true" != "$(SCOPE_SKIP_UI_ASSETS)"; then`) more readable, but maybe that's just me :)